public/pages/AnomalyCharts/containers/FeatureBreakDown.tsx

```diff
@@ -80,10 +83,19 @@ export const FeatureBreakDown = React.memo((props: FeatureBreakDownProps) => {
         const filteredFeatureData = [];
         for (let i = 0; i < anomaliesFound.length; i++) {
           const currentAnomalyData = anomaliesResult.anomalies[i];
+          const dataEntityList = get(
```

Comment:
What's the difference between `dataEntityList` and `cellEntityList`?

Reply:
`dataEntityList` is the associated entity list with the current anomaly data in the `for` loop. `cellEntityList` is the entity list pulled from the selected heatmap cell. Note that no logic changes were made here. Instead, `entityList` used to be `get(currentAnomalyData, 'entity', [] as EntityData[])[0].value` and `cellEntityList` used to be `props.selectedHeatmapCell.entityValue`.

Reply:
> cellEntityList is the entity list pulled from the selected heatmap cell

Does that mean user can choose multiple heatmap cells?

Reply:
No, only one can be selected at a time. But the cell itself can contain a list of entities if it is a multi-category detector